webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -77,18 +77,17 @@ export default class TicketPopover extends React.PureComponent<Props, State> {
         return null;
     };
 
-    componentDidUpdate() {
+    componentDidMount() {
```

Comment:
Does that mean we query the API on every mouseover?

Reply:
Yes, plugins can implement caching on their side to avoid repeated API calls. However, from the webapp performance perspective, this component will get unmounted and remounted.

Reply:
Does the JIRA plugin implement such caching? In other words, does the tooltip load as quickly as before the webapp before on the 2nd mouseover?

Reply:
It loads depending on the network request resolution.

Reply:
Ticket created:  https://mattermost.atlassian.net/browse/MM-62571